bigquery/main_test.go

```diff
@@ -369,28 +385,41 @@ func TestEnsureFunctions(t *testing.T) {
 }
 
 func TestSendNotification(t *testing.T) {
-	cfg := &notifiers.Config{
-		Spec: &notifiers.Spec{
-			Notification: &notifiers.Notification{
-				Filter: `build.build_trigger_id == "1234" `,
-				Delivery: map[string]interface{}{
-					"table": "projects/project_name/datasets/dataset_name/tables/table_name",
-				},
-			},
-		},
-	}
+
 	for _, tc := range []struct {
 		name    string
+		cfg     *notifiers.Config
 		build   *cbpb.Build
 		wantErr bool
 	}{{
-		name: "mising IDs",
+		name: "missing IDs",
+		cfg: &notifiers.Config{
+			Spec: &notifiers.Spec{
+				Notification: &notifiers.Notification{
+					Filter: `build.build_trigger_id == "1234" `,
+					Delivery: map[string]interface{}{
+						"table": "projects/project_name/datasets/dataset_name/tables/valid",
+					},
+				},
+			},
+		},
 		build: &cbpb.Build{
 			BuildTriggerId: "1234",
+			Id:             "1",
 		},
 		wantErr: true,
 	}, {
 		name: "unknown status",
+		cfg: &notifiers.Config{
```

Comment:
Up to you: since you're repeating this config setup, consider making a `validConfig` variable.

Reply:
An alternative is to have a private `makeConfig()` function that takes a minimal number of fields to produce the variety of configs you need.